sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -422,7 +422,7 @@ object FunctionRegistry {
     expression[BitwiseXor]("^"),
 
     // json
-    expression[StructToJson]("to_json"),
+    expression[StructOrArrayToJson]("to_json"),
```

Comment:
I think that one is not particually better. `StructsToJson` then does not refer it can be an array or a single struct if the reason is only ambiguosity. Either way is fine to me. If any committer picks up one, let me follow.